windows/src/engine/testhost/testhost.cpp

```diff
@@ -298,7 +299,9 @@ Fail(PCWSTR message) {
   std::vector<wchar_t> err(bufferLength);
   std::wstring buf;
   DWORD dwError = GetLastError();
-  if (FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM, nullptr, dwError, 0, err.data(), bufferLength, nullptr) != 0) {
+  if (dwError == ERROR_SUCCESS) {
+    buf = std::wstring(message); // Don't append "The operation completed successfully."
```

Comment:
```suggestion
    buf = std::wstring(message) + L": no error code was returned"; // Don't append "The operation completed successfully."
```

Not that it matters very much for this particular in-house program, but I think for clarity we should be at least saying that we got a zero error code.